### PR TITLE
Cleaning geom series

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/lss_examples.rst [hunk 1]
```diff
@@ -40,9 +40,9 @@ Assume that
 
 -  :math:`\{\epsilon_{t}\}` is a sequence of independently and
    identically distributed :math:`N(0,1)` random variables, i.e., mean
-   zero, variance one.
+   zero, variance one
 
--  $:raw-latex:`\sigma `:raw-latex:`\geq `0 $ is a "volatility"
+-  :math:`\sigma` :math:`\geq` 0 is a "volatility"
```
Comment (jstac) [inline source/rst/lss_examples.rst]: This should be :math:`\sigma \geq 0`

**Review by jstac — CHANGES_REQUESTED** [review]

**Comment by jstac** [comment]
@AnjuJoon Just the small change mentioned above please.

We need to capitalize titles as well.  You can do that in a separate PR if that's easier.

File: source/rst/geom_series.rst [hunk 2]
```diff
@@ -619,20 +617,20 @@ below
     get_ipython().run_line_magic('matplotlib', 'inline')
     
     #true present value of a finite lease
-    def finite_lease_pv(T,g,r,x_0):
+    def finite_lease_pv(T, g, r, x_0):
       G=(1+g)
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: Can we add spaces around the equals signs here? We just don't need spaces around = in function arguments.

File: source/rst/geom_series.rst [hunk 3]
```diff
@@ -619,20 +617,20 @@ below
...
       R=(1+r)
       return (x_0*(1-G**(T+1)*R**(-T-1)))/(1-G*R**(-1))
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: Could you also add spaces around operators (ie. +, i, but not **)

File: source/rst/geom_series.rst [hunk 4]
```diff
@@ -642,39 +640,39 @@ Now that we have test run our functions, we can plot some outcomes
...
+However, holding :math:`g` and r fixed, our approximations deteriorate as :math:`T` increases
 
 Next we compare the infinite and finite duration lease present values
 over different lease lengths :math:`T`
 
-.. code:: ipython3
+.. code-block:: python3
 
-    #convergence of infinite and finite. 
+    #convergence of infinite and finite
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: Could you add a space following the # and capitalise the start of comments

File: source/rst/geom_series.rst [hunk 5]
```diff
@@ -768,30 +766,30 @@ represents our present value formula for an infinite lease
...
 
-.. code:: ipython3
+.. code-block:: python3
 
     import sympy as sp
     from sympy import init_printing
     
     #creates algebraic symbols that can be used in an algebraic expression
-    g,r,x0=sp.symbols('g,r,x0')
+    g,r,x0=sp.symbols('g, r, x0')
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: Can you add spaces following the commas on the left side as well?

File: source/rst/geom_series.rst [hunk 6]
```diff
@@ -768,30 +766,30 @@ represents our present value formula for an infinite lease
...
 
     print('dp0/dg is:')
-    dp_dg= sp.diff(p0,g)
+    dp_dg= sp.diff(p0, g)
     dp_dg
 
-.. code:: ipython3
+.. code-block:: python3
 
     print('dp0/dr is:')
```
Comment (natashawatkins) [inline source/rst/geom_series.rst:791]: we can use string formatting to do this neater, ie.

```
print(f'dp0/dr is: {sp.diff(p0, r)}')
```

**Comment by natashawatkins** [comment]
You'll also need to add the same code at the start of the lecture like in #15, and add the lecture to the toctree

File: source/rst/geom_series.rst [hunk 7]
```diff
@@ -856,24 +855,24 @@ i.e., the fraction of income that is consumed
     plt.title('Changing Consumption as a Fraction of Income')
     plt.ylabel('$y_t$')
     plt.xlabel('$t$')
-    for b in (b_0, b_1, b_2,b_3):
-        plt.plot(np.arange(0,T+1),calculate_y(i_0,b,g_0,T,y_init),label= r'$b=$'+f"{b:.2f}")
+    for b in (b_0, b_1, b_2, b_3):
+        plt.plot(np.arange(0, T+1), calculate_y(i_0, b, g_0, T, y_init), label= r'$b=$'+f"{b:.2f}")
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: There shouldn't be a space following `label= ` here.

Also I think this can be simplified to `label=rf'$b = {b:.2f}$'`

**Comment by natashawatkins** [comment]
Your changes from lss have also come through in the PR

**Comment by AnjuJoon** [comment]
@natashawatkins we appended already existing lecture so the standard code in beginning is already there and it is also there in the index file

**Comment by natashawatkins** [comment]
Ah ok you are right!

**Comment by jstac** [comment]
@natashawatkins Are you happy with this now?  Please let us know.

File: source/rst/geom_series.rst [hunk 8]
```diff
@@ -800,96 +799,96 @@ this equation will always be negative. Similarly,
...
-        y=np.zeros(T+1)
-        y[0]=i+b*y_init+g
-        for t in range(1,T+1):
-            y[t]=b*y[t-1]+i+g
+    # Function that calculates a path of y
+    def calculate_y(i, b, g, T, y_init):
+        y = np.zeros(T+1)
+        y[0] = i + b*y_init + g
+        for t in range(1, T+1):
+            y[t] = b*y[t-1] + i + g
```
Comment (natashawatkins) [inline source/rst/geom_series.rst]: `b * y` and above

File: source/rst/lss_examples.rst (hunk 1 above)
Comment (natashawatkins) [reply source/rst/lss_examples.rst]: This still needs fixing

File: source/rst/lss_examples.rst [hunk 9]
```diff
@@ -97,15 +97,16 @@ as a linear state space model in the following way:
...
-long does it take for :math:`Y_t` to reach its stationary distribution?
+We want to ask the following questions of this model: 
+
+\* Does :math:`Y_t` have a stationary distribution? If so, what is it? 
+\* How long does it take for :math:`Y_t` to reach its stationary distribution?
 \* What is the effect of an investment shock, :math:`\sigma \epsilon_t`,
 on present and future values of :math:`C_t,I_t` and :math:`Y_t`?
 
-The methods of the Linear State Space class will be able to help us out.
+The methods of the Linear State Space class will be able to help us out
```
Comment (natashawatkins) [inline source/rst/lss_examples.rst]: `LinearStateSpace` class

File: source/rst/lss_examples.rst (hunk 1 above)
Comment (natashawatkins) [reply source/rst/lss_examples.rst]: Actually this is in another file -- can @AnjuJoon figure out how to reverse changes to this file so we don't merge in any changes here?

**Comment by natashawatkins** [comment]
Looks good! I have one minor change above. I'm worried about the other files which were accidentally included in this PR - could they be reversed?

**Comment by AnjuJoon** [comment]
@natashawatkins  Yes I will reverse those changes (in other file)

File: source/rst/geom_series.rst (hunk 8 above)
Comment (AnjuJoon) [reply source/rst/geom_series.rst]: space in * operator as well?

Comment (jstac) [reply source/rst/geom_series.rst]: Yes.  All operators except `**`

**Comment by natashawatkins** [comment]
@AnjuJoon I've just reversed the changes so they don't appear in the commit. I did this by looking at `git log` and using `git checkout <HASH> -- <FILE>` to reverse any changes made in the branch to the other files.

**Comment by natashawatkins** [comment]
Ah ok the above created merge conflicts somehow...

**Comment by natashawatkins** [comment]
Ok I think the best solution is going to be to delete this PR and open a new one with just the changes to geom_series

**Pull request closed without merging**